domain/objects/obj_file_test.go

```diff
@@ -0,0 +1,57 @@
+package objects
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/TudorHulban/GoLayouter/app/helpers/helpers"
+)
+
+func TestCreateFile(t *testing.T) {
+	fileName := "file.go"
```

Comment:
set a constant and inject it directly in the object

Reply:
addressed in https://github.com/TudorHulban/GoLayouter/commit/3383f1a94dc26557ce25920ece2bf590e6b6dc84